docs/index.md

```diff
@@ -1,8 +1,8 @@
 (polaris)=
 # Polaris
-```{image} _static/polaris_logo_rect.png
+```{image} _static/polaris_logo_large.png
```

Comment:
I just noticed that this linked to `large`, not `small`.  I think it should have been small.  We should fix this when we have time because `large` is more than 1 MB.

Reply:
Sure, can do. Would you like me to make a new PR for this now or wait to roll it into another PR?

Reply:
Thanks, either is fine.